samplecode/sqlite3/enclave/Enclave.edl

```diff
@@ -32,27 +32,6 @@ enclave {
         void ocall_print_error([in, string] const char *str);
         void ocall_print_string([in, string] const char *str);
         void ocall_println_string([in, string] const char *str);
-        
-        int ocall_lstat([in, string] const char *path, [in, out, size=size] struct stat *buf, size_t size) propagate_errno;
-        int ocall_stat([in, string] const char *path, [in, out, size=size] struct stat *buf, size_t size);
-        int ocall_fstat(int fd, [in, out, size=size] struct stat* buf, size_t size);
-        int ocall_ftruncate(int fd, off_t length);
-
-        char* ocall_getcwd([out, size=size] char *buf, size_t size) propagate_errno;
-        int ocall_getpid(void);
-        int ocall_getuid(void);
-        char* ocall_getenv([in, string] const char *name);
-
-        int ocall_open64([in, string] const char *filename, int flags, mode_t mode);
-        int ocall_close(int fd);
-        off_t ocall_lseek64(int fd, off_t offset, int whence) propagate_errno;
-        int ocall_read(int fd, [out, size=count] void *buf, size_t count) propagate_errno;
-        int ocall_write(int fd, [in, size=count] const void *buf, size_t count) propagate_errno;
-        int ocall_fsync(int fd);
-        int ocall_fcntl(int fd, int cmd, [in, size=size] void* arg, size_t size) propagate_errno;
-        int ocall_unlink([in, string] const char *pathname);
-        int ocall_fchmod(int fd, mode_t mode);
-        int ocall_fchown(int fd, uid_t owner, gid_t group);
```

Comment:
Clean up all the ocall syscalls for security concern mentioned by @mssun. Although only support in-memory SQLite database now.